lib/agent.js

```diff
@@ -98,7 +98,7 @@ Agent.prototype.setTransactionName = function (name) {
   return this._instrumentation.setTransactionName.apply(this._instrumentation, arguments)
 }
 
-Agent.prototype.startSpan = function (name, type, { childOf } = {}) {
+Agent.prototype.startSpan = function (name, type, subtype, action, { childOf } = {}) {
```

Comment:
Remember to update the TypeScript typings as well

Reply:
fixed